app/Modules/Api/V1/Model/EventApiService.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace App\Modules\Api\V1\Model;
+
+use App\Modules\Core\Model\Entity\Event;
+use App\Modules\Core\Model\EventModel;
+use Nette\Application\LinkGenerator;
+use Nette\Utils\DateTime;
+
+class EventApiService
```

Comment:
I recommend approach [`Final` by default](https://ocramius.github.io/blog/when-to-declare-classes-final/)